contrib/babelfishpg_tsql/src/pl_exec.c

```diff
@@ -336,7 +336,7 @@ void pltsql_estate_setup(PLtsql_execstate *estate,
 					 EState *simple_eval_estate);
 void exec_eval_cleanup(PLtsql_execstate *estate);
 
-static int exec_fmtonly(PLtsql_execstate *estate,
+int exec_fmtonly(PLtsql_execstate *estate,
```

Comment:
Where is this function actually being used?

Reply:
This function is not getting use anywhere that's why I have remove the static instead of removing the function so that compiler don't show warning for function not getting used.

Reply:
Tim said to don't remove the `exec_fmtonly` function as it is temporarily disabled until some workarounds get added.